nym-vpn-x/src-tauri/src/main.rs

```diff
@@ -201,7 +201,8 @@ async fn main() -> Result<()> {
                 }
             });
 
-            let handle = app.handle();
+            // TODO check if clone works
+            let handle = app.handle().clone();
```

Comment:
It does and some objects like Tray and others have a method to obtain it via `app_handle()` too, which may come very handy.

Reply:
> some objects like Tray and others have a method to obtain it via app_handle() too, which may come very handy

yep I know

Indeed, in tauri v1, idk why but it was not needed to clone the handle ref (and taking owership) see https://github.com/nymtech/nym-vpn-client/blob/45297614d96c3e0b27bf9ccf0bd08215e7009ffb/nym-vpn-x/src-tauri/src/main.rs#L204

now in tauri v2 something as changed.
```
error[E0521]: borrowed data escapes outside of closure
   --> src/main.rs:179:26
    |
157 |         .setup(move |app| {
    |                      ---
    |                      |
    |                      `app` is a reference that is only valid in the closure body
    |                      has type `&'1 mut tauri::App`
...
179 |             let handle = app.handle();
    |                          ^^^^^^^^^^^^
    |                          |
    |                          `app` escapes the closure body here
    |                          argument requires that `'1` must outlive `'static`
```

And at first glance it looked suspicious so I left few TODO comments, but yea it seems to work as expected, so I'll remove them.